### DISCUSSION TITLE
社区参与

### DISCUSSION THREAD

**Discussion by zlmk321** [halo-dev/halo#5565] [General]
### 你当前使用的版本

2.13.1

### 描述一下此特性

建议增加社区参与的机会，例如开设论坛、提供技术支持、组织活动等，以促进社区建设和贡献和增强用户互动和分享体验

**Comment by ruibaby** [comment]
与此仓库本身无关，我将转移到 Discussion。

/triage unresolved

**Comment by ruibaby** [comment]
- 官方社区：https://bbs.halo.run/
- 社区例会：https://bbs.halo.run/d/3919-halo-%E7%A4%BE%E5%8C%BA%E4%BE%8B%E4%BC%9A%E5%90%88%E9%9B%86%E9%A2%84%E5%91%8A%E4%B8%8E%E5%9B%9E%E6%94%BE/16
